src/patternfly/components/Table/table.scss

```diff
@@ -806,15 +789,28 @@
   }
 }
 
-// - Table action - Table favorite - Table inline edit action - Table draggable
+// - Table action - Table favorite - Table inline edit action - Table draggable - Table sort - Table toggle
 .#{$table}__action,
 .#{$table}__favorite,
 .#{$table}__inline-edit-action,
-.#{$table}__draggable {
-  --#{$table}--cell--PaddingBlockStart: var(--#{$table}__action--PaddingBlockStart);
-  --#{$table}--cell--PaddingBlockEnd: var(--#{$table}__action--PaddingBlockEnd);
-  --#{$table}--cell--PaddingInlineStart: var(--#{$table}__action--PaddingInlineStart);
-  --#{$table}--cell--PaddingInlineEnd: var(--#{$table}__action--PaddingInlineEnd);
+.#{$table}__draggable,
+.#{$table}__sort,
+.#{$table}__toggle {
+  // set tbody actionable cells padding-block-start to zero, as negative margins impact row height
+  .#{$table}__thead & {
+    padding-block-start: 0;
+    padding-block-end: var(--#{$table}--cell--action--PaddingBlockEnd);
+  }
+
+  // set tbody actionable cells padding-block-start to zero, as negative margins impact row height
+  .#{$table}__tbody & {
+    padding-block-start: var(--#{$table}--cell--action--PaddingBlockStart);
+    padding-block-end: 0;
+  }
```

Comment:
For `tbody` `.#{$table}--cell--action` cells, only bottom padding needs to be applied in order to create consistent vertical alignment, as `tbody` is set to `vertical-align: top`.